pkg/gvk/gvk.go

```diff
@@ -290,3 +325,29 @@ func isCRDGroup(group string, crdGroups []string) bool {
 
 	return false
 }
+
+// sortResources sources resources by moving extensions to the end of the slice. The order of all
+// the other resources is preserved.
+func sortResources(resources []*metav1.APIResourceList) {
```

Comment:
Another piece borrowed from Velero to accurately simulate the logic used in Velero to determine which resources should be prioritized in a backup.